FirebaseAuth/FirebaseAuth.gd

```diff
@@ -1,13 +1,18 @@
 extends HTTPRequest
 
 signal login_succeeded(auth_result)
-#warning-ignore:unused_signal
 signal login_failed
+signal userdata_received(userdata)
 
-onready var API_Key = ""
-onready var signup_request_url = "https://www.googleapis.com/identitytoolkit/v3/relyingparty/signupNewUser?key="
-onready var signin_request_url = "https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword?key="
-onready var refresh_request_url = "https://securetoken.googleapis.com/v1/token?key="
+var API_Key = ""
+var signup_request_url = "https://www.googleapis.com/identitytoolkit/v3/relyingparty/signupNewUser?key="
+var signin_request_url = "https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword?key="
+var userdata_request_url = "https://www.googleapis.com/identitytoolkit/v3/relyingparty/getAccountInfo?key="
+var refresh_request_url = "https://securetoken.googleapis.com/v1/token?key="
+
+const REPONSE_SIGNIN   = "identitytoolkit#VerifyPasswordResponse"
```

Comment:
Would you mind fixing the typo here and make sure these are all "RESPONSE" instead of "REPONSE"?